ui/helpers/constants/settings.js

```diff
@@ -287,6 +287,16 @@ export const SETTINGS_CONSTANTS = [
     route: `${SECURITY_ROUTE}#security-alerts-opensea`,
     icon: 'fa fa-lock',
   },
+  ///: BEGIN:ONLY_INCLUDE_IF(keyring-snaps)
```

Comment:
Wrong code fence?

Reply:
busteeeed

Reply:
Co-pilot was not doing his part, sorry :D